frontend/src/components/_shared/DashboardDatasetCard.tsx

```diff
@@ -16,119 +27,105 @@ export interface DashboardDatasetCardProps {
   contributors: { imageUrl: string }[];
 }
 
-export default ({
-  tdcValidated,
-  title,
-  keywords,
-  visibility,
-  dateState,
-  updateFrequency,
-  region,
-  variant,
-  href,
-  contributors,
-}: DashboardDatasetCardProps) => {
+export default function DashboardDatasetCard(props: Dataset) {
   const router = useRouter();
+
+  const {
+    id,
+    name,
+    tdc_category,
+    title,
+    tags,
+    metadata_modified,
+    frequency,
+    state,
+    geographies,
+    contributors,
+    organization,
+  } = props;
+
+  const badgeVariant =
+    tdc_category === "tdc_harmonized"
+      ? "warning"
+      : tdc_category === "tdc_formatted"
+      ? "success"
+      : "purple";
+
+  const badgeIconOptions = {
+    width: 20,
+    height: 20,
+  };
+
+  const badgeIcon =
+    tdc_category === "tdc_harmonized" ? (
+      <ShieldCheckIcon {...badgeIconOptions} />
+    ) : tdc_category === "tdc_formatted" ? (
+      <CheckCircleIcon {...badgeIconOptions} />
+    ) : (
+      <CircleStackIcon {...badgeIconOptions} />
+    );
+
+  const { data: contributorsData, isLoading } = api.user.getUsersByIds.useQuery(
+    {
+      ids: contributors,
+    }
+  );
+
   return (
-    <div
-      onClick={() => router.push(href)}
-      className="flex w-full cursor-pointer gap-6"
+    <Link
+      href={`/@${organization?.name}/${id}`}
```

Comment:
Remove this @ of the URI it works as organization.name/id

Reply:
its working fine with @. should I removed it? I see its a common practice in other portals.

Reply:
Nvm, it works both ways:

https://tdc-data-portal.vercel.app/@Transport%20Data%20Commons/7064aa02-91a9-43a0-97e8-89da0a173f12
https://tdc-data-portal.vercel.app/Transport%20Data%20Commons/7064aa02-91a9-43a0-97e8-89da0a173f12